src/components/Checkbox/index.module.scss

```diff
@@ -0,0 +1,51 @@
+@import "../styles/foundation.scss";
+
+$size: 18px;
+$borderRadius: 2px;
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: start;
+  cursor: pointer;
+  padding: 3px 4px;
+  border-radius: 8px;
+
+  .rectangle {
+    width: $size;
+    min-width: $size;
+    height: $size;
+    display: flex;
+    border-radius: $borderRadius;
+    border-width: $defaultBorder;
+    border-style: solid;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    transition: background-color $tranzition150;
+
+    input[type="checkbox"] {
+      display: none;
+    }
+  }
+
+  .content {
+    position: relative;
+    padding-left: 16px;
+    font-family: Roboto;
```

Comment:
You do not need use font property you have classes for it in .srotybook/preview.js